### ISSUE TITLE
Invalid URI error: arrival-and-departure-for-stop

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-deep-links#19]
Stack:

```
lib/oba/server.rb:21 in arrival_and_departure
app/controllers/alarms_controller.rb:8 in create
```

Error:

```
bad URI(is not URI?): http://api.tampa.onebusaway.org/api/api/where/arrival-and-departure-for-stop/Hillsborough Area Regional Transit_7690.json?app_uid=C071187D-67E0-458C-A1DA-CADE062AE667&app_ver=20170105.12&key=org.onebusaway.iphone&serviceDate=1488258000000&stopSequence=6&tripId=Hillsborough+Area+Regional+Transit_234879&vehicleId=Hillsborough+Area+Regional+Transit_2603&version=2
```

**Comment by sayhiben** [comment]
Duplicate of #7?

**Comment by aaronbrethorst** [comment]
@sayhiben Fixed by the URL escaping fix that'll work for #7, but I want to add a separate test for this to validate that it keeps working properly.

**Comment by aaronbrethorst** [comment]
I'm going to close this and I'll deal with the other issue appropriately.
